classes/controller/menu/class-en-settings-controller.php

```diff
@@ -18,8 +22,12 @@ class En_Settings_Controller extends Controller {
 	 * Create menu/submenu entry.
 	 */
 	public function create_admin_menu() {
+		// We need to check if the migration already ran, as the EN block is on by default, but we cannot give an option
+		// that was added using CMB2 a default value of 'on', because then it can't be turned off.
+		$migration_ran     = MigrationLog::from_wp_options()->already_ran( M001EnableEnFormFeature::get_id() );
```

Comment:
I guess we can remove it once this migration passes ? let's keep a post-it or a ticket or something